### ISSUE TITLE
examples/random_token_example, when I run python distill.py then exception Killed

### ISSUE THREAD

**Issue by dulante00** [airaria/TextBrewer#79]
intermediate_matches : [
IntermediateMatch: layer_T : 0, layer_S : 0, feature : hidden, weight : 1, loss : hidden_mse, proj : None,
IntermediateMatch: layer_T : 8, layer_S : 2, feature : hidden, weight : 1, loss : hidden_mse, proj : None,
IntermediateMatch: layer_T : [0, 0], layer_S : [0, 0], feature : hidden, weight : 1, loss : nst, proj : None,
IntermediateMatch: layer_T : [8, 8], layer_S : [2, 2], feature : hidden, weight : 1, loss : nst, proj : None]
is_caching_logits : False

  0%|                                                                                                                                                                                       | 0/10 [00:00<?, ?it/s]
0it [00:00, ?it/s]

Killed
